martin/src/cog/source.rs

```diff
@@ -23,6 +23,7 @@ struct Meta {
     min_zoom: u8,
     max_zoom: u8,
     model: ModelInfo,
+    origin: [f64; 3],
```

Comment:
Could you add doc comments explaining what the origin is?
This would be very handy as origin as 3*f64 is not quite self-explanatory.

Reply:
Hmm. I will try.  Actually it's the coords: raster_space_to_model_space(0,0,0), I'm not good at this, maybe you have better one? 
I add this to:
```rust
    // The geo coords of pixel(0, 0, 0) ordering in [x, y, z]
    origin: [f64; 3],
```